test/unittests/commands/xmlupload/test_ontology_client.py

```diff
@@ -176,10 +176,10 @@ def test_get_ontology_names_from_server() -> None:
             "logo": None,
             "longname": "Bernoulli-Euler Online",
             "ontologies": [
-                "http://www.knora.org/ontology/0801/biblio",
-                "http://www.knora.org/ontology/0801/newton",
-                "http://www.knora.org/ontology/0801/leibniz",
-                "http://www.knora.org/ontology/0801/beol",
+                "http://api.dasch.swiss/ontology/0801/biblio/v2",
+                "http://api.dasch.swiss/ontology/0801/newton/v2",
+                "http://api.dasch.swiss/ontology/0801/leibniz/v2",
+                "http://api.dasch.swiss/ontology/0801/beol/v2",
```

Comment:
When I send the request to a real API from a project I get the ontology as follows:

```
    "ontologies": [
            "http://www.knora.org/ontology/002A/church"
        ],
```

Reply:
It is a breaking change in DSP-API, so it depends on what server you use.

The following servers still have the old DSP-API version:
 - `dsp-tools start-stack` (on main branch)
 - prod
 - all RDU test servers

And the following servers have the new DSP-API version:
 - most of the dev servers
 - test.dasch.swiss 
 - stage.dasch.swiss